src/components/Pagination.jsx

```diff
@@ -46,6 +46,10 @@ const Icon = styled.img`
 `;
 
 const Pagination = ({ currentPage, totalPages, onPageChange }) => {
+  // 페이지네이션 범위 계산 (5개씩)
+  const rangeStart = Math.floor((currentPage - 1) / 5) * 5 + 1;
+  const rangeEnd = Math.min(rangeStart + 4, totalPages); // 최대 5개의 페이지 번호만 보여주기
```

Comment:
매직넘버의 사용을 지양하고(5) 상수로 관리하는게 좋을것 같습니다!
```suggestion
  const PAGE_VIEW = 5;
  //  직접 구현해보세요!
```